src/topology/builder.rs

```diff
@@ -229,7 +230,7 @@ pub async fn build_pieces(
                             Ok(TaskOutput::Healthcheck)
                         }
                         Ok(Err(error)) => {
-                            error!(message = "Healthcheck: Failed Reason.", %error);
+                            error!(message = "Healthcheck: Failed Reason.", %component_name, %error);
```

Comment:
Maybe we could add `component_type` here too? It would just be hardcoded to `sink`, but I think that'd make it easier for people to understand what is happening if they don't already know healthchecks are only applicable to sinks.

Reply:
Actually it is `component_kind` that should be `sink`, I've added both `component_kind` and `component_type`.

Reply:
Ah, yes, thank you!